docs(processes): upper vulnerable version policy (#549)
nodejs/security-wg: processes/third_party_vuln_process.md

If the maintainers are unreachable, the vulnerability is to be made public
45 days after the triage date.

> The report's upper limit of the vulnerable versions should be set to:

Reviewer: ```suggestion
The report's vulnerable versions upper limit should be set to:
```

> * `*` if there is no fixed version available by the time of publishing the report. This is mostly in cases where the maintainer hasn't been able to take part in the triage process.

Reviewer: ```suggestion
* `*` if there is no fixed version available by the time of publishing the report. This is mostly in cases where the maintainer hasn't taken part in the triage process.
```

> * the version less than that which a fix exists for. For example: `<=1.2.3` if a fix exists in `1.2.4`

Reviewer: ```suggestion
* the last vulnerable version. For example: `<=1.2.3` if a fix exists in `1.2.4`
```
